pkg/email/templates/issue_lcid.gohtml

```diff
@@ -1,6 +1,7 @@
 <p>Lifecycle ID: {{.LifecycleID}}</p>
 <p>Expiration Date: {{.ExpiresAt}}</p>
 <p>Scope: <pre style="white-space: pre-wrap; word-break: keep-all;">{{.Scope}}</pre></p>
+<p>Project Cost Baseline: <pre style="white-space: pre-wrap; word-break: keep-all;">{{.LifecycleCostBaseline}}</pre></p>
```

Comment:
You should check if the cost baseline is not null here because it is not required, you can just use the format that next steps uses below.